helm/kamaji/values.yaml

```diff
@@ -37,8 +37,11 @@ etcd:
       name: root-client-certs
       # -- Name of the namespace where the secret which contains ETCD client certificates is. (default: "kamaji-system")
       namespace: kamaji-system
-    # -- (string) Comma-separated list of the endpoints of the etcd cluster's members.
-    endpoints: "https://etcd-0.etcd.kamaji-system.svc.cluster.local:2379,https://etcd-1.etcd.kamaji-system.svc.cluster.local:2379,https://etcd-2.etcd.kamaji-system.svc.cluster.local:2379"
+    # -- (map) Dictionary of the endpoints for the etcd cluster's members, key is the name of the etcd server
+    endpoints:
+      etcd-0: https://etcd-0.etcd.kamaji-system.svc.cluster.local:2379
```

Comment:
etcd.endpoints are used in the controller as following:

```yaml
      - args:
        - --config-file=./kamaji.yaml
        - --etcd-ca-secret-name=etcd-certs
        - --etcd-ca-secret-namespace=kamaji-system
        - --etcd-client-secret-name=root-client-certs
        - --etcd-client-secret-namespace=kamaji-system
        - --etcd-compaction-interval=0
        - --etcd-endpoints=etcd-0=https://etcd-0.etcd.kamaji-system.svc.cluster.local:2379,etcd-1=https://etcd-1.etcd.kamaji-system.svc.cluster.local:2379,etcd-2=https://etcd-2.etcd.kamaji-system.svc.cluster.local:2379
        - --health-probe-bind-address=:8081
        - --leader-elect
        - --metrics-bind-address=:8080
        - --tmp-directory=/tmp/kamaji
        command:
        - /manager
        image: clastix/kamaji:latest
```

With this PR we're going to set as `--etcd-endpoints=etcd-0=https://etcd-0.etcd.kamaji-system.svc.cluster.local:2379,etcd-1=https://etcd-1.etcd.kamaji-system.svc.cluster.local:2379,etcd-2=https://etcd-2.etcd.kamaji-system.svc.cluster.local:2379` leading to the following error:

`\"transport: Error while dialing dial tcp: address etcd-2=https://etcd-2.etcd.kamaji-system.svc.cluster.local:2379: too many colons in address\""`